lib/UdoitUtils.php

```diff
@@ -274,6 +274,15 @@ public function sortReportGroups($report_groups)
         return $ordered_report_groups;
     }
 
+    public function checkMultitenant($domain)
+    {
+        $result = UdoitDB::query('SELECT * FROM institutes WHERE domain = "'.$domain.'"')->fetchObject();
```

Comment:
Use $db_institutes_table here instead of a constant to allow for installations where table names require prefixes or other special naming conventions.

Reply:
Also suggest using PDO's prepared statements along with bindParam or bindValue to reduce sql injection vectors

see: https://github.com/ucfopen/UDOIT/blob/36a659a5693e3c26a34966ce75023642b57614a9/lib/UdoitUtils.php#L92-L94